templates/zerver/app/keyboard_shortcuts.html

```diff
@@ -186,6 +186,10 @@
                     <td class="hotkey">Esc, Ctrl + [</td>
                     <td class="definition">{% trans %}Narrow to all unmuted messages{% endtrans %}</td>
                 </tr>
+                <tr>
+                    <td class="hotkey">Ctrl + .</td>
+                    <td class="definition">{% trans %}Narrow to compose target{% endtrans %}</td>
```

Comment:
I think we want this to say "Narrow to current compose box recipient"

@rishig in case you have a better proposal.

Reply:
that sounds good to me